config/production.json

```diff
@@ -123,6 +123,7 @@
 		"onboarding/import-from-wordpress": true,
 		"onboarding/import-redirect-to-themes": true,
 		"onboarding/interval-dropdown": true,
+		"onboarding/new-user-survey": true,
```

Comment:
Not sure if it's accidental, but I'd expect this flag to not enable by default in all the environment, at least not in production, staging, and wpcalypso, so that we don't have to remove them before shipping :)  The purpose should be just a simpler way for the stakeholders to test, so we can just disable it in all environment, and let folks test by adding the query string `?flags=onboarding/new-user-survey` in this case.